pattern-library/sass/patterns/_icons.scss

```diff
@@ -6,6 +6,119 @@
 // NOTE: Read more at https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Icons
 
 // ----------------------------
+// #CONFIG
+// ----------------------------
+$edx-icons: linkedin-square
```

Comment:
This is nice abstraction, but two nits: 
- Can you put "linkedin-square" on it's own line like the rest?
- Is there anyway to grab these from the source so we don't have to update it when a new one is added? Or is this our custom list?